cpp/velox/operators/functions/RegistrationAllFunctions.cc

```diff
@@ -39,12 +39,14 @@ void registerFunctionOverwrite() {
   velox::exec::registerStatefulVectorFunction(
       "murmur3hash",
       velox::functions::sparksql::hashWithSeedSignatures(),
-      velox::functions::sparksql::makeHashWithSeed);
+      velox::functions::sparksql::makeHashWithSeed,
+      velox::functions::sparksql::hashMetadata());
```

Comment:
Seems no need to register murmur3hash?

I note there is a mapping here:
https://github.com/apache/incubator-gluten/blob/main/cpp/velox/substrait/SubstraitParser.cc#L401

Reply:
Right. Let me remove the xxhash64 registration as well and add mapping `xxhash64 -> xxhash64_with_seed` here.